modules/apps/transfer/keeper/migrations_test.go

```diff
@@ -194,8 +194,8 @@ func (suite *KeeperTestSuite) TestMigrateTotalEscrowForDenom() {
 				coin2 := sdk.NewCoin(sdk.DefaultBondDenom, sdk.NewInt(100))
 
 				// funds the escrow accounts to have balance
-				suite.Require().NoError(banktestutil.FundAccount(suite.chainA.GetSimApp().BankKeeper, suite.chainA.GetContext(), escrowAddress1, sdk.NewCoins(coin1)))
-				suite.Require().NoError(banktestutil.FundAccount(suite.chainA.GetSimApp().BankKeeper, suite.chainA.GetContext(), escrowAddress2, sdk.NewCoins(coin2)))
+				suite.Require().NoError(banktestutil.FundAccount(suite.chainA.GetContext(), suite.chainA.GetSimApp().BankKeeper, escrowAddress1, sdk.NewCoins(coin1)))
```

Comment:
Here's another spot where I think I got things right but I'm not sure @tac0turtle

Reply:
what is the question here?

Reply:
I'm sorry, on second glance I think that this was not the correct thing.  I will mark it again when I reencounter and thanks a ton for your reply!